docs/modules/ROOT/examples/js/kamelets.js

```diff
@@ -16,6 +16,8 @@
  */
 
 const util = require('camel-website-util')
+const fs = require('fs')
+const yaml = require('js-yaml');
```

Comment:
If you're importing here you also need to adjust package.json in camel-website, i.e. add `js-yaml` to `devDependencies`. You also need to make sure that the `js-yaml` is placed in the PnP cache. Or avoid using `js-yaml` or any non-managed dependency.

I can see the net positive of having them versioned within the module. But I don't see a way for them to be tested, debugged or linted here. So until this is figured out I can't recommend doing this here.

Reply:
Do we have to revert? The website build is stuck

Reply:
Ops, I will push the js-yaml dependency change to camel-website. Just forgot it.